config/default.json

```diff
@@ -1,19 +1,18 @@
 {
-  "server": {
-    "port": 3009
-  },
-  "fileSystem": {
-    "workflowsPath": "./workflows"
-  },
-  "validator": {
-    "workflowFields": ["name", "activities"],
-    "dynamicActivityFields": ["action", "method", "additional"],
-    "dynamicActivityNameValues": ["apiInvoker"],
-    "ingestFields": ["imageMetaData", "action"]
-  },
-  "logger": {
-    "level": "info",
-    "log2file": true
-  }
+    "server": {
+        "port": 3000
```

Comment:
delete file and add to git ignore since its generated

Reply:
fixed in 9943996